docker/Dockerfile

```
@@ -16,12 +16,13 @@ RUN apt-get update && apt-get install -y ffmpeg libsm6 libxext6 git ninja-build
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-# Install MMCV MMDetection
-RUN pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6.0/index.html
-RUN pip install mmdet
+# Install openmim
+RUN pip install --no-cache-dir -U openmim
+# Install mmengine, mmcv, and mmdetection
+RUN mim install --no-cache-dir mmengine "mmcv>=2.0.0rc2" "mmdet>=3.0.0rc2"
 # Install MMRotate
-RUN conda clean --all
-RUN git clone https://github.com/open-mmlab/mmrotate.git /mmrotate
+RUN conda clean --all -y
+RUN git clone https://github.com/open-mmlab/mmrotate.git -b dev-1.x /mmrotate
```

```suggestion
RUN git clone https://github.com/open-mmlab/mmrotate.git -b 1.x /mmrotate
```
Since the previous line targeted the `'main'` branch, how about setting the branch to `'1.x'`?

---

@nijkah  I agree it.